meta/main.yml

```diff
@@ -1,10 +1,8 @@
 ---
 galaxy_info:
-  role_name: k8s-ansible
+  role_name: k8s
   description: >
-    This role install Kubernetes using kubeadm with OpenStack 
-    This role provisions an exisiting cluster to support Beegfs
-    management, metadata, object storage server and client roles.
+    This role install Kubernetes using kubeadm with OpenStack.
   license: Apache2
```

Comment:
Actually, it is not OpenStack specific, unless you set `cloud_provider: openstack`